### ISSUE TITLE
Updating DBeaver when installed as a snap package creates two independent versions of DBeaver

### ISSUE THREAD

**Issue by nrgsourcecode** [dbeaver/dbeaver#10912]
#### System information: 
- Operating system (distribution) and version: Ubuntu 20.04
- DBeaver version: 7.3.1

#### Describe the problem you're observing:
I installed dbeaver-ce through Ubuntu Software, which installed the snap package. When I got a notification that a new version is available, I clicked on the the update button which conveniently downloaded a deb package file, but after that I ended up with two versions of dbeaver, one from the snap package, and the other one as a standalone package.

**Comment by kseniiaguzeeva** [comment]
It is not implemented yet. Marked as Feature Request.

**Comment by jaweed-khan** [comment]
Same issue appeared in Ubuntu 20.04 
Any suggestions on how to update the dbeaver on ubuntu after we install through snap ?

**Comment by sastorsl** [comment]
Is there really a fix for this?
If you have installed DBeaver through snap I should think you would have to update the package through snap aswell, meaning wait until a new snap version is available.

**Comment by arnoudhgz** [comment]
Yeah, quite annoying that I get a notification that an update is available, but `snap refresh` gives me `snap "dbeaver-ce" has no updates available`.

**Comment by swpalmer-cl** [comment]
Even more annoying that the link to the snap is provided on the download page https://dbeaver.io/download/ along side the other packages, yet it is for an older version.  If you are going to update the packages - update all of them.
Or at least show the version number beside the download links if they aren't all the same. The download page clearly has ONE version number at the top, so it is assumed all the links are for that version.
